tests/ci/run_check.py

```diff
@@ -223,12 +224,14 @@ def check_pr_description(pr_info) -> Tuple[str, str]:
     if pr_labels_to_remove:
         remove_labels(gh, pr_info, pr_labels_to_remove)
 
-    commit.create_status(
-        context="Simple Check",
-        description="Skipped",
-        state="success",
-        target_url=GITHUB_RUN_URL,
-    )
+    rerun_helper = RerunHelper(gh, pr_info, "Simple Check")
```

Comment:
We need a function in commit_status_helper that would check the current status and mark as success only by current green